Makefile

```diff
@@ -1,16 +1,47 @@
-# this make target is designed to be ran idempotently. each run deploys the latest code in the repository
-# requires kubernetes. both minikube and docker desktop is supported.
+# Copyright 2019 clair authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+COMMIT = $(shell git describe --tag --always --dirty)
+
+UNIT_TEST_PACKAGES = $(shell go list ./... | grep -v 'database/')
+DB_TEST_PACKAGES = $(shell go list ./... | grep 'database/')
+
+.PHONY: build
+build:
+	go build -v -ldflags "-X github.com/coreos/clair/pkg/version.Version=$(COMMIT)" ./cmd/clair
+
+.PHONY: unit-test
+unit-test:
+	go test $(UNIT_TEST_PACKAGES)
+
+.PHONY: db-test
+db-test:
```

Comment:
@jzelinskie 

Are these integration tests which require a DB to be available ?

Reply:
Yup! Should I make a doc comment clarifying as much?

Reply:
@jzelinskie yup! comments would be good. One other thing on my mind is *techincally* we can get these to work with the local dev setup, tho I'm assuming you added this for CI and will call it in the Drone container it's going to run in? 

I do not feel strongly either way, just food for thought since we can launch postgres with local dev and make the postgres db available locally.